addons/web/models/models.py

```diff
@@ -1099,7 +1103,7 @@ def __init__(self, record: BaseModel, fields_spec: dict, fetch=True):
             for name in fields_spec:
                 self.fetch(name)
 
-    def __eq__(self, other: 'RecordSnapshot'):
+    def __eq__(self, other: RecordSnapshot):
```

Comment:
This has nothing to do with the commit about `NewId`.

Reply:
I probably did this on a wrong branch. Removed.